test/Vm.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.5;
```

Comment:
[Not a comment on this PR but on foundry]Foundry should be injecting this interface by default or it should provide a path to import it.